meta-coav/LICENSE

```diff
@@ -0,0 +1,54 @@
+Apache License
```

Comment:
LICENSE check in the recipe should be against the file that was fetched with the code ( to confirm that project's LICENSE didn't change). So you shouldn't ship this file in the layer and check against it, wouldn't make much sense because the check would always be true.

The LICENSE file to be used should be the one on the root level of our repository.